edx_proctoring/backends/backend.py

```diff
@@ -54,6 +54,13 @@ def mark_erroneous_exam_attempt(self, exam, attempt):
         """
         raise NotImplementedError()
 
+    @abc.abstractmethod
+    def remove_exam_attempt(self, exam, attempt):
+        """
+        Method that removes the exam attempt from the backend's system
```

Comment:
It could say that, but the returned status isn't checked, which begs the question about what should happen in the event that the backend returns False.

Reply:
I don't understand. The returned status isn't checked by what? In your implementation of this method in `rest.py`, the status is being checked.

Reply:
It's not checked [here](https://github.com/edx/edx-proctoring/pull/539/files#diff-8475abe05c644ffd4d41872d58426b5cR114) because I'm not clear what should happen if there's a failure.

Reply:
I see. I personally feel it would be better not to return anything if the returned value is not being used. Are you choosing to return whether an attempt is successfully deleted because it's a common paradigm for deletion to attempt a deletion and then return whether or not it's successful?